ledger-core/pulsewatcher/runner.go

```diff
@@ -25,15 +25,12 @@ import (
 	"github.com/insolar/assured-ledger/ledger-core/pulsewatcher/status"
 )
 
+var client http.Client
```

Comment:
lets avoid global variable here and just create one client in Run and pass it to CollectNodesStatuses?

Reply:
ok